src/app/components/header/header.component.html

```diff
@@ -0,0 +1,22 @@
+  <mat-toolbar color="primary">
+  <div fxHide.gt-xs>
+      <button mat-icon-button (click)="onToggleSidenav()">
+          <mat-icon>menu</mat-icon>
```

Comment:
Looks like mat-icon isn't working. I think we need to import the icons in the index.html file. Let me know if you need some help here

Reply:
done